### PR TITLE
Feature: add onEventLongPress prop

### PR DESCRIPTION
**Pull request by pdpino**
Add `onEventLongPress` prop, to allow more user interactions with the events

### CODE REVIEW

**Comment by hoangnm** [comment]
Hi @pdpino , could I know the use case of event long press? I'm just curious.

**Comment by hoangnm** [comment]
nvm, I just saw the latest comments from the mentioned issue.

**Comment by pdpino** [comment]
That's a good question.

As part of the edit-mode feature discussed in #58, I was thinking to add a prop to allow the developer enabling edit-mode _on-press_ or _on-long-press_ (see [this comment](https://github.com/hoangnm/react-native-week-view/issues/58#issuecomment-847256964)).
So, if the developer chooses to enable edit-mode with _press_, they may want to use the _on-long-press_ for something else (e.g. showing a modal with event info).

In other words:
* Developer A: enables edit-mode with **on-long-press**, and uses **on-press** to do something with the event.
* Developer B: enables edit-mode with **on-press**, and uses **on-long-press** to do something with the event.

@hoangnm let me know what you think! If you think this is not necessary, or any other comments. :slightly_smiling_face:

**Comment by hoangnm** [comment]
Thanks @pdpino , I think it's nice to have this.

**Comment by rdewolff** [comment]
> That's a good question.
> 
> As part of the edit-mode feature discussed in #58, I was thinking to add a prop to allow the developer enabling edit-mode _on-press_ or _on-long-press_ (see [this comment](https://github.com/hoangnm/react-native-week-view/issues/58#issuecomment-847256964)).
> So, if the developer chooses to enable edit-mode with _press_, they may want to use the _on-long-press_ for something else (e.g. showing a modal with event info).
> 
> In other words:
> 
> * Developer A: enables edit-mode with **on-long-press**, and uses **on-press** to do something with the event.
> * Developer B: enables edit-mode with **on-press**, and uses **on-long-press** to do something with the event.
> 
> @hoangnm let me know what you think! If you think this is not necessary, or any other comments. 🙂

This seems like a great idea. Enabling “edit mode” we could animate the meetings with something like a little “shake” to show the user that the events can be dragged.
